data_steward/utils/participant_summary_requests.py

```diff
@@ -393,3 +450,53 @@ def store_participant_data(df, project_id, destination_table, schema=None):
     job.result()
 
     return job.job_id
+
+
+def store_digital_health_status_data(project_id,
+                                     json_data,
+                                     destination_table,
+                                     schema=None):
```

Comment:
Sure, we can discuss this over a call. This is helpful when using `records` which cannot fit into a dataframe easily. The data is uploaded in JSON format since it is one of the ways BQ expects it.